### PR TITLE
Update the Eucalyptus upgrade instructions.

### PR DESCRIPTION
**Pull request by nedbat**
Just a start on the Eucalyptus updates.

### CODE REVIEW

**Comment by nedbat** [comment]
@lamagnifica look ok?

File: en_us/install_operations/source/platform_releases/eucalyptus.rst [hunk 1]
```diff
@@ -155,9 +155,9 @@ Automatic Upgrading
 `The upgrade.sh script`_ is in the edX configuration repository on GitHub.
 
 .. note::
-  The upgrade scripts provided are verified only for upgrading instances
-  running the Dogwood release. If you are running any other version of the Open
-  edX Platform, the upgrade scripts might not work.
+  The upgrade script is only for upgrading instances running the Dogwood
+  release. If you are running any other release of the Open edX Platform,
```
Comment (lamagnifica) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst]: lc platform

File: en_us/install_operations/source/platform_releases/eucalyptus.rst [hunk 2]
```diff
@@ -155,9 +155,9 @@ Automatic Upgrading
...
+  follow the instructions to upgrade them to each next successive release.
```
Comment (lamagnifica) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst]: strike "next"
not sure what "them" refers to -- the instances?
Maybe something like,
If your instance is running a release prior to the Dogwood release, follow the instructions to upgrade it to each intervening release, and then upgrade from Dogwood to Eucalyptus.

File: en_us/install_operations/source/platform_releases/eucalyptus.rst [hunk 3]
```diff
@@ -167,11 +167,21 @@ Automatic Upgrading
 On the computer or virtual machine that is running the Dogwood release of Open
 edX, run the upgrade script for your type of installation.
 
-* For devstack, run
-  ``./upgrade.sh -c devstack -t open-release/eucalyptus/latest``.
+1.  Download the script.
```
Comment (lamagnifica) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst]: #. should work instead of the digits (you fixed it!)

Comment (lamagnifica) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst]: maybe add the xref here paraphrasing line 155: "Download `the upgrade.sh script`_ from the edX configuration repository on GitHub."

File: en_us/install_operations/source/platform_releases/eucalyptus.rst [hunk 4]
```diff
@@ -167,11 +167,21 @@ Automatic Upgrading
...
 
-* For fullstack, run
-  ``./upgrade.sh -c fullstack -t open-release/eucalyptus/latest``.
+    .. code-block:: bash
```
Comment (lamagnifica) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst:173]: wait, is this a second step, or is this the command that downloads the script? (if the latter, ignore my previous comment)

File: en_us/links/links.rst [hunk 5]
```diff
@@ -144,7 +144,7 @@
 
 .. _edx configuration repository wiki: https://github.com/edx/configuration/wiki
 
-.. _The migrate.sh script: https://github.com/edx/configuration/blob/master/util/vagrant/migrate.sh
+.. _The migrate.sh script: https://github.com/edx/configuration/blob/named-release/dogwood.rc/util/vagrant/migrate.sh
```
Comment (lamagnifica) [inline en_us/links/links.rst:147]: I have a bug to fix the dogwood "migrate.sh" script to "upgrade.sh"
Perhaps these anchors should include the release name? 
I will make a separate PR to fix, but propose these two anchors:
- _Dogwood upgrade.sh script:
- _Eucalyptus upgrade.sh script:

Comment (lamagnifica) [inline en_us/links/links.rst:147]: DOC-3218

**Comment by lamagnifica** [comment]
thanks @nedbat a few suggestions 👍

File: en_us/install_operations/source/platform_releases/eucalyptus.rst (hunk 4 above)
Comment (nedbat) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst:173]: This is not a second step, it's the commands to download the script.
I wish I could use a colon to clearly tie the two together.... :(

File: en_us/links/links.rst (hunk 5 above)
Comment (nedbat) [inline en_us/links/links.rst:147]: Well, the dogwood instructions could still reference migrate.sh, on the dogwood branch.  That's what this change does.

Comment (lamagnifica) [inline en_us/links/links.rst:147]: I think you'll run into trouble if you have dogwood instruction AND Eucalyptus instructions in the same guide and they both use the same anchor. Don't you get a build error?

Comment (lamagnifica) [inline en_us/links/links.rst:147]: wait, I'm so confused. Is there a valid Dogwood migrate.sh script in addition to the upgrade.sh script?

**Comment by lamagnifica** [comment]
dandy!

**Pull request merged**